Sources/Sharing/SharedPublisher.swift

```diff
@@ -15,7 +15,9 @@
     /// }
     /// ```
     public var publisher: some Publisher<Value, Never> {
-      box.subject.prepend(wrappedValue)
+      box.subject
+        .handleEvents(receiveSubscription: { [box] _ in _ = box })
```

Comment:
This seems like a sloppy way to capture the reference...not sure if there's a better way.